polaris-tokens/src/token-groups/spacing.ts

```diff
@@ -51,3 +51,5 @@ export const spacing = {
     value: '128px',
   },
 };
+
+export type SpacingTokens = keyof typeof spacing;
```

Comment:
```suggestion
export type SpacingTokenName = keyof typeof spacing;
```

Our current model of a `token` is an object with a `value` and `description` field. To avoid confusion, I suggest updating to use a more explicit naming convention.